docker-configuration/docker-compose.yml

```diff
@@ -44,6 +45,7 @@ services:
       - ENV DB_USERNAME=root
       - ENV DB_PASSWORD=password
       - ENV DB_URL=jdbc:postgresql://posts-db:5432/postsdb
+      - ENV USER_URL=jdbc:postgresql://posts-db:5432/postsdb
```

Comment:
A little remark: it is better not to glue "jdbc:..", port, and db name directly in the docker-compose file, it is better to do it at the application level. The db port can be passed via env